MC/config/QC/json/trd-digits-task.json

```diff
@@ -9,70 +9,139 @@
         "name": "not_applicable"
       },
       "Activity": {
-        "number": "42",
+        "number": "505618",
         "type": "2",
-        "provenance": "qc_mc",
-        "passName": "passMC",
-        "periodName": "SimChallenge"
+        "periodName": "",           "": "Period name - e.g. LHC22c, LHC22c1b_test",
+        "passName": "",             "": "Pass type - e.g. spass, cpass1",
+        "provenance": "qc",         "": "Provenance - qc or qc_mc depending whether it is normal data or monte carlo data"
       },
       "monitoring": {
         "url": "infologger:///debug?qc"
       },
       "consul": {
-        "url": "http://localhost:8500"
+        "url": "http://consul-test.cern.ch:8500"
       },
       "conditionDB": {
         "url": "ccdb-test.cern.ch:8080"
+      },
+      "infologger": { "": "Configuration of the Infologger (optional).",
+        "filterDiscardDebug": "false",    "": "Set to 1 to discard debug and trace messages (default: false)",
+        "filterDiscardLevel": "2", "": "Message at this level or above are discarded (default: 21 - Trace)" 
       }
     },
     "tasks": {
-      "QcTask": {
+      "RawDataTask": {
+        "active": "false",
+        "className": "o2::quality_control_modules::trd::RawData",
+        "moduleName": "QcTRD",
+        "detectorName": "TRD",
+        "cycleDurationSeconds": "60",
+        "maxNumberCycles": "-1",
+        "dataSource_comment": "no comment",
+        "dataSource": {
+          "type": "direct",
+          "query": "digits:TRD/DIGITS;tracklets:TRD/TRACKLETS;triggers:TRD/TRKTRGRD;rawstats:TRD/RAWSTATS"
```

Comment:
Are you sure that `rawstats:TRD/RAWSTATS` is accesible at this stage of computation (or in simulated data at all)?
Here is what TRD QC requests to be present:
https://github.com/AliceO2Group/O2DPG/blob/ca1971d3835f9f5186f488993281e3960fecdd95/MC/bin/o2dpg_sim_workflow.py#L785
Here is the documentation about adding QC to MC scripts:
https://github.com/AliceO2Group/O2DPG/tree/master/MC#adding-qc-tasks-to-the-simulation-script

Reply:
rawstats would not work no, and also the tracklet task is removed for not wanting to edit the python.